topics/genome-annotation/tutorials/secondary-metabolite-discovery/tutorial.md

```diff
@@ -0,0 +1,391 @@
+---
+layout: tutorial_hands_on
+draft: true
```

Comment:
this should be enough to make it invisible for most, what will be the link for the draft ?

Reply:
@paulzierep yep, this is all that is needed. The url will be the same as it would be otherwise, it is just hidden from the topic page.

If you ever can't find the link, we have this new "stellar nursery" page that lists all the tutorials currently in draft mode: https://training.galaxyproject.org/utils/stellar-nursery